features/steps/topic_component.go

```diff
@@ -74,6 +81,42 @@ func NewTopicComponent(mongoFeature *componenttest.MongoFeature, zebedeeURL stri
 	return f, nil
 }
 
+func createCredsInDB(getMongoURI string, databaseName string) (string, string) {
+	username := "admin"
+	password, _ := uuid.NewV4()
+	mongoConnectionConfig := &dpMongoDriver.MongoConnectionConfig{
+		IsSSL:                   false,
+		ConnectTimeoutInSeconds: 15,
+		QueryTimeoutInSeconds:   15,
+
+		Username:        "",
+		Password:        "",
+		ClusterEndpoint: getMongoURI,
+		Database:        databaseName,
+	}
+	mongoConnection, _ := dpMongoDriver.Open(mongoConnectionConfig)
```

Comment:
Do you need to check this error? Presumably fail the test if this is not nil.

Reply:
updated in the next commit.